backend/python/app/services/implementations/story_service.py

```diff
@@ -703,6 +707,37 @@ def finish_grading_story_translation(
                 .all()
             )
 
+            translate_passed = test_result["translate"]
+            review_passed = test_result["review"]
+
+            if translate_passed or review_passed:
+                language = story_translation_test.language
+                level = (
```

Comment:
ah shoot. I should've been more specific, the new level does not have to come from the story_test level. I.e, the admins have requested to be able to assign higher translation/review levels if they subjectively feel a test was really outstanding.  I think it makes sense for test_result to look like `"translate": 4, "review": 3` where 4 and 3 are the new levels accordingly. see attched modal to get a sense of that flow, lmk if you have questions!
![image](https://user-images.githubusercontent.com/15113249/146659207-dc8468be-da84-4327-a959-b15cc97cdcba.png)